main.go

```diff
@@ -258,6 +262,37 @@ func ShowVersion() {
 	os.Stdout.Write([]byte(v))
 }
 
+func doBootstrapAuthorize(config *menderConfig, opts *runOptionsType) error {
+	store := NewDirStore(*opts.dataStore)
+
+	authreq, err := NewAuthClient(config.GetHttpConfig())
+	if err != nil {
+		return errors.New("cannot auth client")
```

Comment:
Maybe include the original `err` in the error here, so you know what the reason is?